### DISCUSSION TITLE
Would you consider a Filament Plugin?

### DISCUSSION THREAD

**Discussion by idaraty** [coderflexx/laravisit#34] [Ideas]
Great work ! congratulations.
I'm wondering if you would consider creating a plugin for [Filament PHP](https://filamentphp.com/). That would be awesome !
Thank you for your consideration

**Comment by ousid** [comment]
Hey, @idaraty 

Sorry for the late response.

Yea, sure. Sounds a perfect idea. I'll try to work on it once I have time.

__PS__: Feel free to shoot ideas on how you imagine being done this exactly.

Thanks for the suggestion!
